components/compositing/touch.rs

```diff
@@ -45,7 +47,15 @@ pub enum TouchState {
     /// Contains the initial touch location.
     Touching,
     /// A single touch point is active and has started panning.
-    Panning,
+    Panning {
+        velocity: Vector2D<f32, DevicePixel>,
+    },
+    /// No active touch points, but there is still scrolling velocity
+    /// Note: For now we only support flinging along the y-axis
```

Comment:
I think this comment can be removed now?

```suggestion
    /// Note: For now we only support flinging along the y-axis
```